src/lib.rs

```diff
@@ -57,10 +57,10 @@ use std::ptr;
 use std::sync::RwLock;
 
 lazy_static! {
-    // We do not use "new" directly here because it can fail and we do not want to panic. The
+    // At the moment of writing, this would only fail with a wrongly configured endpoint. The
     // automatic selection of authentication/provider is done in `p_init` so that we can return an
     // error if there is a problem with that.
-    static ref PARSEC_BASIC_CLIENT: RwLock<BasicClient> = RwLock::new(BasicClient::new_naked());
+    static ref PARSEC_BASIC_CLIENT: RwLock<BasicClient> = RwLock::new(BasicClient::new_naked()?);
```

Comment:
Can we squash this change with the other commit ?

Reply:
I think your comment is marked as "outdated". Is this still valid? If so, can you specify which commits?

Reply:
Its from this [commit](https://github.com/parallaxsecond/parsec-se-driver/pull/36/commits/ea8b797dee86a7d2aacf90a062570ea0238b0870)

Reply:
Done